### PR TITLE
mod - updating required mobile version.

### PR DESCRIPTION
**Pull request by mebassett**
this will force yoroi into maint mode for all users until we are
happy with a shelley-enabled version.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -241,7 +241,7 @@ const txBodies = async (req: Request, res: Response) => {
 const getStatus = async (req: Request, res:  Response) => {
   const mobilePlatformVersionPrefixes = ["android / ", "ios / ", "- /"];
   const clientVersionHeader = 'yoroi-version';
-  const minMobileVersion = '2.2.2';
+  const minMobileVersion = '2.2.3';
```
Comment (nicarq) [inline src/index.ts:244]: @mebassett I think this should be 2.2.4, so 2.2.3 is still shown in maintenance

**Comment by nicarq** [comment]
@mebassett this will work for mobile. For Desktop are you thinking of doing a new PR?

File: src/index.ts (hunk 1 above)
Comment (SebastienGllmt) [reply src/index.ts:244]: yeah either that or we can check the sermver check to be <= instead of <

**Comment by mebassett** [comment]
> @mebassett this will work for mobile. For Desktop are you thinking of doing a new PR?

no, this won't work for desktop.  actually that's trickier.  for me, my production chrome extension is sending `yoroi-version: chrome / 0.0.0`.   I'd prefer to resolve that in this PR.  can someone confirm that later versions of yoroi are sending out a non-zero string? 

(it doesn't really matter, because a quick semverCompare would still put it into maintenance mode.  but I also want to be able to come out of maint mode.)

**Comment by SebastienGllmt** [comment]
Ah yes, this is because a very old bug in Yoroi where Yoroi instead sends the version you last opened Yoroi with instead of the version you're currently using. That means it sends 0.0.0 the very first time you launch Yoroi.

**Comment by mebassett** [comment]
> Ah yes, this is because a very old bug in Yoroi where Yoroi instead sends the version you last opened Yoroi with instead of the version you're currently using. That means it sends 0.0.0 the very first time you launch Yoroi.

what should we do about this?  It does not sound as though its a big deal, as the app will go into maint mode now and come out of it after they upgrade (eventually...)   Should we ignore it for this PR

can you confirm the desired version number for the desktop extensions?  The version I'm seeing in my production extension (after the `0.0.0`) is `1.10.3` but locally with a development build I'm getting `3.0.0`.

**Comment by SebastienGllmt** [comment]
The next version we're releasing is 3.0.0
Yes, it doesn't matter if 0.0.0 is set to maintenance since we'll remove the maintenance check after the new version is deployed to all users.

File: src/index.ts [hunk 2]
```diff
@@ -240,8 +240,10 @@ const txBodies = async (req: Request, res: Response) => {
 
 const getStatus = async (req: Request, res:  Response) => {
   const mobilePlatformVersionPrefixes = ["android / ", "ios / ", "- /"];
+  const desktopPlatformVersionPrefixes = ["firefox / ", "chrome / "];
   const clientVersionHeader = 'yoroi-version';
-  const minMobileVersion = '2.2.2';
+  const minMobileVersion = '2.2.4';
+  const minDesktopVersion = '1.10.4';
```
Comment (mebassett) [inline src/index.ts:246]: should this be `3.0.0` then?

Comment (SebastienGllmt) [reply src/index.ts:246]: either `1.10.4` or `3.0.0` should work equally fine.

Comment (mebassett) [reply src/index.ts:246]: okay.  anything else you'd like to see before merging-and-deploying ?

**Pull request merged**